sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/variant/VariantExpressionEvalUtils.scala

```diff
@@ -58,9 +59,13 @@ object VariantExpressionEvalUtils {
       false
     } else {
       val variantValue = input.getValue
-      // Variant NULL is denoted by basic_type == 0 and val_header == 0
-      variantValue(0) == 0
-     }
+      if(variantValue.isEmpty) {
```

Comment:
I guess this fails at `Scala` linter? Could you double-check this, @harshmotw-db ?

Reply:
Resolved